src/index.html

```diff
@@ -10,13 +10,116 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__container">
+        <div class="header__logo">
+          <a
+            href="#moyo"
+            class="header__logo-link"
+          >
+            <img
+              src="images/logo.png"
+              alt="moyo logo"
```

Comment:
The `alt` attribute for the image should describe the content of the image. Instead of 'moyo logo', it would be better to describe what the logo looks like or what it represents.